installer/bundle/bundle_skel.sh

```diff
@@ -930,7 +917,7 @@ esac
 
 # Remove temporary files (now part of cleanup_and_exit) and exit
 
-if [ "$KIT_STATUS" -ne 0 -o "$BUNDLE_EXIT_STATUS" -ne 0 ]; then
+if [ "$SCX_STATUS" -ne 0 -o "$KIT_STATUS" -ne 0 -o "$BUNDLE_EXIT_STATUS" -ne 0 ]; then
     cleanup_and_exit 1
```

Comment:
So if SCX failed, we do return an error. I guess the only question is: Is it worth trying to do the other things before returning the error? I'm not sure, but we should think about this.